orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/persistence/ExecutionRepository.groovy

```diff
@@ -32,6 +32,8 @@ interface ExecutionRepository {
   void cancel(String id)
   void cancel(String id, String user)
 
+  void forceCancel(String id, String user)
```

Comment:
why do we have a cancel and forceCancel? Shouldn't forceCancel just be the default behavior?